packages/manager/modules/manager-pci-common/src/components/quantity-selector/QuantitySelector.component.tsx

```diff
@@ -61,7 +62,7 @@ export function QuantitySelector({
   useEffect(forceRedraw, [min, max]);
 
   return (
-    <OsdsFormField className={className} inline>
+    <OsdsFormField className={className} inline {...props}>
```

Comment:
could you not put directly the test-id on the QuantitySelector component ? why do you have to add it to the inner form field ?

Reply:
What i am trying to do is the total opposite of what you said, the main gaol of this is to avoid having test-id directly in the component, did you see a test-id on the inner field?the wrapper of the quantity selector is a Formfield, if it's mocked, how would i differentiate them?